versioned_docs/version-8.1/components/connectors/out-of-the-box-connectors/amazon-eventbridge.md

```diff
@@ -44,8 +44,8 @@ Follow these steps to configure the Amazon EventBridge Connector:
    - **Source**: Enter the value that identifies the service that generated the event.
    - **Detail type**: Enter the type of event being sent. Refer to the [Amazon documentation](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-events-structure.html) for more information on these properties.
 5. In the **Event Payload** section, enter a JSON object that contains information about the event.
-6. (Optional) In the **Output Mapping** section, you can set a **Result variable** or **Result expression**. Refer to the [response mapping documentation](/docs/components/connectors/use-connectors/index.md#response-mapping) to learn more.
-7. (Optional) In the **Error Handling** section, define the **Error expression** to handle errors that may occur during the event sending process. Refer to the [response mapping documentation](/docs/components/connectors/use-connectors/index.md#bpmn-errors) to learn more.
+6. (Optional) In the **Output Mapping** section, you can set a **Result variable** or **Result expression**. Refer to the [response mapping documentation](components/connectors/use-connectors/index.md#variableresponse-mapping) to learn more.
```

Comment:
The links updated in this PR seem to be missing the first `/`. I _think_ you only want to remove `/docs`.

Reply:
I intentionally removed `/docs/` including slashes on both sides, because I associate markdown links that begin with a `/` as typically referencing `/docs`, and ending in a cross-instance link (which these don't do, but it's muscle memory I guess). I don't really have a preference since both resolve to the correct place. Is this feedback blocking?

Reply:
Not blocking, just wanted to make sure it was intentional since other links in these same files are `/components/` vs. `components/`